src/boot/efi/stub.c

```diff
@@ -76,10 +75,17 @@ EFI_STATUS efi_main(EFI_HANDLE image, EFI_SYSTEM_TABLE *sys_table) {
                 cmdline = line;
```

Comment:
(hmm, the above line is a sloppy version of an utf16-to-utf8 conversion. We should really fix that one day and have a proper one. i.e. we have utf8_to_16(), we should also have the reverse... but anywy, this is probably material for a different PR.)